Formula/projectm.rb

```diff
@@ -0,0 +1,24 @@
+class Projectm < Formula
+  desc "Milkdrop-compatible music visualizer"
+  homepage "https://github.com/projectM-visualizer/projectm"
+  url "https://github.com/projectM-visualizer/projectm/releases/download/v3.1.11/projectM-3.1.11.tar.gz"
+  sha256 "4f60506637d178b96646bdd3c78161f6e30a89dee556675625d6e03dea0ee791"
+  license "LGPL-2.1-or-later"
+  head "https://github.com/projectM-visualizer/projectm.git"
+
+  depends_on "sdl2" => :build
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system bin/"projectM-unittest"
```

Comment:
Is this something a user would regularly run?

Reply:
It's not no